vault/token_store.go

```diff
@@ -3208,6 +3208,10 @@ func (ts *TokenStore) tokenStoreRoleRead(ctx context.Context, req *logical.Reque
 		resp.Data["token_num_uses"] = role.TokenNumUses
 	}
 
+	if role.TokenNoDefaultPolicy {
+		resp.Data["token_no_default_policy"] = true
```

Comment:
Is there a reason not to return `false` if it is not set?

Reply:
not really, tbh. I wasn't sure of the criteria on which some fields are always in the response, and some only in the response if they've been set by user input. Since this is currently omitted, I erred on the side of the change that affects existing behavior least, so went with only returning this value if set and keeping the response as is otherwise.

Reply:
I would add it.

Reply:
Roger, that sounds good to me too, will add it in 👍

Reply:
6e1ba28